testing/metaopt/test_automl_sequential_wrapper.py

```diff
@@ -38,7 +40,7 @@ def test_automl_sequential_wrapper(tmpdir):
 
     # Then
     actual_mse = ((predicted_outputs - expected_outputs) ** 2).mean()
-    assert actual_mse < 5000
+    assert actual_mse < 20000
```

Comment:
resolve to issue: use something better than RandomSearch to not have flickering tests.

Reply:
https://github.com/Neuraxio/Neuraxle/issues/289